content/post/34-go-ipfs-0.4.14.md

```diff
@@ -2,7 +2,7 @@
 date: 2018-03-21
 url: 34-go-ipfs-0.4.14
 title: go-ipfs 0.4.14 released
-author: Victor Bjelkholm
+author: @whyrusleeping & @victorbjelkholm
```

Comment:
<3 I wasnt gonna say anything since you went through the trouble to set this up

Reply:
Heh, yeah, but I can't take all the credit! I'm questioning if I can even take any, but I am going to anyways.